pymc/distributions/continuous.py

```diff
@@ -1456,12 +1457,11 @@ class Exponential(PositiveContinuous):
 
     Notes
     -----
-    Logp calculation is defined in aeppl.logprob
-    https://github.com/aesara-devs/aeppl/blob/main/aeppl/logprob.py
+    Logp calculation is defined in `aeppl.logprob <https://github.com/aesara-devs/aeppl/blob/main/aeppl/logprob.py/>`_.
```

Comment:
This is an improvement but ideally that would use sphinx cross references cc @ricardoV94. Are there plans to document the functions in logprob module? 

related also to https://github.com/pymc-devs/pymc/issues/5329